packages/fluentui/react-bindings/src/context-selector/useContextSelectors.ts

```diff
@@ -104,5 +104,5 @@ export const useContextSelectors = <
     };
   }, [listeners]);
 
-  return state[1] as SelectedValue;
+  return state[1] as Record<Properties, SelectedValue>;
```

Comment:
Similarly, I managed to build without this `as` cast

Reply:
```
src/context-selector/useContextSelectors.ts:107:3 - error TS2322: Type 'SelectedValue' is not assignable to type 'Record<Properties, SelectedValue>'.
  Type 'unknown' is not assignable to type 'Record<Properties, SelectedValue>'.
```

Tried locally, it breaks 😪